### ISSUE TITLE
People do not build bases before playing with the mods

### ISSUE THREAD

**Issue by stefment** [NillerMedDild/Enigmatica6#864]
**Modpack version**
0.3.5

**Issue**
It seems most people do not build a base before playing with the mods. So they either put the machines etc. on the grass or dig a hole in a mountain and put the stuff there. In my opinion this is unacceptable. People should build a base before being allowed to play with mods imo. I am not sure there is a way to enforce this. But perhaps players could be encouraged to build a base by starting with for example the building gadget (with a free, one time 100% charge). And then maybe an entry in the quest log that yells at players to build at least a basic base with a floor, walls/windows before playing with mods :) Have a nice day

**Comment by stefment** [comment]
Any suggestion for how to encourage players to build a base before starting is welcome :) I am interested in hearing them. I saw a mod that had a starter house bluebrint thing that gives players a basic base to start with. Maybe thats option?

**Comment by Merlo17** [comment]
This is a really weird one tbh. It really does depend on playstyle and what people like doing most, doesn't it? If you like to build cool bases then you totally have all the tools to do so from the start, but you should also be free to just... not do that. The building gadget is super cheap already if someone wants it, and the quests are there to guide the player into discovering the pack, not straight up telling them what they should do, in the end one should be free to just choose their own path (besides the fact that most people already know they should build a base, it's minecraft after all, they just choose not to, and that should be an acceptable playstyle).

Blueprints/copy paste gadgets imo make it even worse, since not only you're not building your own base, but you're copying someone else's. Why should we enforce a certain style of play on players who might not like it? Everything should be fair game here, and it just seems like something that's just personal, and we as pack makers should make everyone's preference valid, not favour one.

**Comment by Ridanisaurus** [comment]
I would rather live in a dirt hole than use Prefab blue prints. Also, surviving couple of nights in the wild, and building in the open is an essential part of the Minecraft experience. Adding mods or tools early that 'forces' players to build before they can play the game is a terrible idea in my opinion. 
Building a dirt house, or digging a hole to survive your first night while gathering the materials required to start a base, or explore to find that sweet biome is a Minecraft 101.

Also, forcing players to do something is a bad mechanic, especially in a sandbox type of game like Minecraft.

**Comment by MuteTiefling** [comment]
A bit of extra perspective here. Enigmatica's general goal is to make a cohesive and welcoming experience for all players. 

Recipes are tweaked to help lift up some mods or bring others in line, not to gate things as you'd see in an Expert version. Quests aim to teach and reward.
 
Similarly, efforts are made to unify stuff across mods so you don't have a million different types of copper you can't use. And so that all those ores from other mods can be used in some way in other places. 

No attempts are made to force any particular form of play style. Sure, we have some mobs that make life a bit more dangerous early on. We have some extra bosses you can choose to fight. But we don't have anything that's meant to punish, like Blood moons of yore, or zombies that break through walls. 

At the end of the day, Minecraft (and Kitchen Sink mod packs like E6) is a sandbox and should encourage creativity in all its forms, not strictly try to enforce one particular type of creativity. And make no mistake, building complex machines with tech is no less a creative endeavor than building fancy buildings. They are two sides of the same coin.

**Comment by stefment** [comment]
What ridanisaurus says is 100% correct. but i believe that most people have been through this surviving a couple nights in the wild and building in the open digging a basic hole to survive, many times, and to go through this again is not why they come to modded. I believe and i could be wrong, that if you put the building gadget into the hands of people as they spawn, they will check it out and probably realise this is pretty neat i can actually build a neat shelter to begin with.

Adding to that, because people go through the having to survive in the wild, this may be why they just end up building not even a barebones base, but just a hole in a mountain, because theyve been through the process of building a base and its not what interest them. They skip it! But by placing the building gadget in their hands as they spawn (i would like to note that they have to gather any materials they wanna build with) by putting the gadget in players hands as they spawn with a free one time 100% charge, most people would find that refreshing! but lets see

**Comment by Merlo17** [comment]
Just because you don't like holes in a mountain doesn't mean nobody should ever be allowed to build them, or that they're wrong for building them though. It's just the way they choose to live life, and I don't see why we should tell them it's invalid. 

Of course, keep going around and building cool bases, as will I and probably many others. In the end it's about doing things you enjoy: it's a game after all

**Comment by stefment** [comment]
What im saying is that digging a hole in the mountain and calling it a day to play with mods is wrong and modders and modpacks should probably do more to encourage base building.

**Comment by Ridanisaurus** [comment]
Wrong, in your definition. I know a lot of people who would spend their entire playthrough inside a hole, and that's fine too. We encourage it by giving them the tools and options, but we can't force them to do anything. 
Like they say, 'You can lead a jackass to the water, but you can't force it to drink.'

Most importantly, and like I just said, you are looking at this from your own perspective, and you can't expect every player to play the way YOU want them to play.

**Comment by KrAzYGEEK32** [comment]
asking the pack devs to encourage base building is atleast some what sensible, but straight up calling other playstyles wrong is.... not ok imo . people play games for different reasons, especially minecraft. for some people building is not an enjoyable part of the game (like me, i love my 9x9s) . and yea as rid said, it might be wrong.. from your perspective, but you should consider other views before calling them wrong.

**Comment by TemperedFool** [comment]
I think it's probably worth tossing out the judgement (aka living in a hole is wrong), since it's opinion and kinda contra the whole kitchen sink concept. 

The potentially worthwhile suggestion of giving starting players a building gadget (notice, not copy/paste gadget) that's charged and ready to go could be slightly more interesting. Here's how I'm thinking about it:

PRO:
- It's a tool that is kinda worth learning, and some newer players may be unaware of it
- It doesn't particularly break any sort of progression (such as there is in a kitchen sink pack)
- Generating with it charged (especially given it has a pretty big power buffer) gets players moving with it immediately, so if they CHOOSE not to live in a hole they can build easier & quickly

CON:
- As Merlo said, the BG is already cheap.
- Very basic power gen isn't hard to get to, so charging isn't a huge gate.
- We need materials to use with it, which works best with tools like bigger Backpacks and at least an intro Dank, meaning the player will likely need to hide in a hole and do some digging before building anyway.

It's a tossup as far as I'm concerned, but I'd probably say don't bother unless you want to set up a few quests leading the player in the direction of the building tools in the pack (which probably isn't necessary).

**Comment by RixMixed** [comment]
Obviously extremely low priority but a single quest or maybe even a chapter that mentions some of the neat tools and resources available for "builders" in the pack could be neat. (Thinking of Embelishcraft, BuildersAdditions, Macaw's, etc.)

**Comment by MuteTiefling** [comment]
> Obviously extremely low priority but a single quest or maybe even a chapter that mentions some of the neat tools and resources available for "builders" in the pack could be neat. (Thinking of Embelishcraft, BuildersAdditions, Macaw's, etc.)

That's planned still, as far as I know. Just has been a bit lower priority than the bigger quest chapters.
